### PR TITLE
BZ-2107179: Doc changes for NHC Operator description

### PR DESCRIPTION
**Pull request by abhatt-rh**
BZ-2107179: This PR updated issues reported in the BZ-2107179 related to the NHC controller

Version(s): OpenShift 4.11+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2107179

Preview:
http://file.rdu.redhat.com/avbhatt/bz-2107179/nodes/nodes/eco-node-health-check-operator.html#about-node-health-check-operator_node-health-check-operator

### CODE REVIEW

**Comment by abhatt-rh** [comment]
Hi @prabinovRedhat and @slintes Based on https://github.com/openshift/openshift-docs/pull/47364#issuecomment-1184399181, Polina created BZ-2107179. This PR addresses those comments. PTAL and approve the changes

**Comment by prabinovRedhat** [comment]
cc: @slintes

File: modules/eco-node-health-check-operator-about.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="about-node-health-check-operator_{context}"]
 = About the Node Health Check Operator
 
-The Node Health Check Operator deploys the `NodeHealthCheck` controller to detect the health of a node in the cluster. The `NodeHealthCheck` controller creates the `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health.
+The Node Health Check Operator detects the health of a node in the cluster. The `NodeHealthCheck` controller creates the `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health.
```
Comment (pneedle-rh) [inline modules/eco-node-health-check-operator-about.adoc:9]: You could change "the node's health" to "the health of a node", to avoid using a possessive.

Comment (pneedle-rh) [reply modules/eco-node-health-check-operator-about.adoc:9]: You could consider changing "detects the health of a node in the cluster" to something like "detects the health of the nodes in a cluster", if it applies to all of the nodes.

**Review by pneedle-rh — COMMENTED** [review]
@abhatt-rh I have completed my review of this PR. I have added a couple of small suggestions. Other than these, the PR looks good to me!

Please let me know if you have any questions about my suggestions. Thanks!

**Comment by slintes** [comment]
lgtm

**Comment by prabinovRedhat** [comment]
lgtm

**Comment by abhatt-rh** [comment]
@pneedle-rh Thank you! I have addressed your comments. PTAL

**Comment by pneedle-rh** [comment]
LGTM!

**Pull request merged**